src/modules/components/member/memberDataListItem/memberDataListItemStructure/memberDataListItemStructure.test.tsx

```diff
@@ -0,0 +1,110 @@
+import '@testing-library/jest-dom';
+import { render, screen } from '@testing-library/react';
+import { getAddress, isAddress } from 'viem';
+import { useAccount, useEnsAddress, useEnsAvatar, useEnsName } from 'wagmi';
+import { DataList } from '../../../../../core';
+import { MemberDataListItemStructure, type IMemberDataListItemProps } from './memberDataListItemStructure';
+
+jest.mock('viem', () => ({
+    isAddress: jest.fn(),
+    getAddress: jest.fn(),
+}));
+
+jest.mock('viem/ens', () => ({
+    normalize: jest.fn(),
+}));
+
+jest.mock('wagmi', () => ({
+    useEnsAddress: jest.fn(),
+    useEnsName: jest.fn(),
+    useEnsAvatar: jest.fn(),
+    useAccount: jest.fn(),
+}));
+
+jest.mock('../../memberAvatar', () => ({ MemberAvatar: () => <div data-testid="member-avatar-mock" /> }));
+
+describe('<MemberDataListItem /> component', () => {
+    const createTestComponent = (props?: Partial<IMemberDataListItemProps>) => {
+        const completeProps: IMemberDataListItemProps = {
+            address: '0x1234567890123456789012345678901234567890',
+            ...props,
+        };
+
+        return (
+            <DataList.Root entityLabel="Members">
+                <DataList.Container>
+                    <MemberDataListItemStructure {...completeProps} />
+                </DataList.Container>{' '}
+            </DataList.Root>
+        );
+    };
+
+    beforeEach(() => {
+        (isAddress as unknown as jest.Mock).mockImplementation(() => true);
+        (getAddress as jest.Mock).mockImplementation((address: string) => address);
+        (useEnsAddress as jest.Mock).mockReturnValue({ data: null, isLoading: false });
+        (useEnsName as jest.Mock).mockReturnValue({ data: null, isLoading: false });
+        (useEnsAvatar as jest.Mock).mockReturnValue({ data: 'mock-avatar-url', isLoading: false });
```

Comment:
This mock should not be needed anymore, I suppose that `useEnsAddress` & `useEnsAvatar` mock can be removed as well. Please check and remove mocks that aren't needed anymore
```suggestion
```